modules/smtp/scanner.go

```diff
@@ -72,30 +73,15 @@ type Flags struct {
 	zgrab2.BaseFlags `group:"Basic Options"`
 	zgrab2.TLSFlags  `group:"TLS Options"`
 
-	// SendEHLO indicates that the EHLO command should be set.
-	SendEHLO bool `long:"send-ehlo" description:"Send the EHLO command; use --ehlo-domain to set a domain."`
-
-	// SendHELO indicates that the HELO command should be set.
-	SendHELO bool `long:"send-helo" description:"Send the HELO command; use --helo-domain to set a domain."`
-
 	// SendHELP indicates that the client should send the HELP command (after HELO/EHLO).
 	SendHELP bool `long:"send-help" description:"Send the HELP command"`
 
 	// SendQUIT indicates that the QUIT command should be set.
 	SendQUIT bool `long:"send-quit" description:"Send the QUIT command before closing."`
 
-	// HELODomain is the domain the client should send in the HELO command.
-	HELODomain string `long:"helo-domain" description:"Set the domain to use with the HELO command. Implies --send-helo."`
-
-	// EHLODomain is the domain the client should send in the EHLO command.
-	EHLODomain string `long:"ehlo-domain" description:"Set the domain to use with the EHLO command. Implies --send-ehlo."`
-
 	// SMTPSecure indicates that the entire transaction should be wrapped in a TLS session.
 	SMTPSecure bool `long:"smtps" description:"Perform a TLS handshake immediately upon connecting."`
 
-	// StartTLS indicates that the client should attempt to update the connection to TLS.
-	StartTLS bool `long:"starttls" description:"Send STARTTLS before negotiating"`
```

Comment:
Do we lose value in being able to do starttls for servers that just do HELO ?
Additionally, I need to do some research, but were we able to determine whether a server SUPPORTS StartTLS vs REQUIRES StartTLS. If so, then it seems like we probably lost that here by autoinvoking it?